k8s-install/scripts/install-cni.sh

```diff
@@ -77,6 +77,11 @@ do
 			continue
 		fi
 		cp $path $dir/
+		if [ "$?" != "0" ];
+		then
+            echo "Failed to copy $path to $dir. This may be caused by selinux configuration on the host, or something else."
```

Comment:
Last thing - intendation looks wrong here. Could you indent this to match?

thanks!

Reply:
sorry for  such mistake